src/violin.jl

```diff
@@ -1,11 +1,22 @@
-
 # ---------------------------------------------------------------------------
-# Violin Plot
```

Comment:
Also disagree with this change - in Plots the style is to just add the utility functions but still keep it all labelled "Violin Plot"